ChangeLog.txt

```diff
@@ -72,6 +74,9 @@ BUGFIXES:
 - Fixed the oversight which made it impossible to keep the Bureaucratic government as a Romance empire after adopting it
 - Fixed Zoroastrian vassals of Persia being feudal government rather than Eransharhr
 - Fixed the Bubonic plague being announced in events without it being on the map
+- Fixed Western Roman Imperial Regalia artifact spawns
+- Fixed part of Italy appearing as being held by Julius Nepos between 476 and 480
```

Comment:
I don't think this one was present in 0.9.10?

Reply:
What do you mean?

Reply:
I don't remember to be honest, but this file hasn't been touched for some time.

Reply:
You're right, the previous version was added on 6 October, so after the 0.9.10.1.

Reply:
I moved this line from fixed to minor and edited singificantly.